### PR TITLE
Merge Cycle into develop

### PR DESCRIPTION
**Pull request by marcperel**
- Moved messaging settings from Customizer to Settings panel in the admin
- Updated settings panel styling (desktop and mobile)
- Implemented front end styling according to the Coil CI
- Fixed compatibility issue with Hemingway theme (and possibly other themes)
- Updated help links in settings panel

### CODE REVIEW

File: src/extensions/components/monetize-mode.js [hunk 1]
```diff
@@ -32,12 +32,12 @@ export class MonetizedMode extends Component {
 
 export default compose( [
 	withSelect( ( select ) => ( {
-		isActive: select( 'core/editor' ).getEditedPostAttribute( 'meta' )[ '_coil_monetize_post_status' ],
```
Comment (JoblersTune) [inline src/extensions/components/monetize-mode.js:35]: Let's leave this as it was.

File: src/extensions/components/monetize-mode.js [hunk 2]
```diff
@@ -32,12 +32,12 @@ export class MonetizedMode extends Component {
...
+		isActive: select( 'core/editor' ).getEditedPostAttribute( 'meta' )._coil_monetize_post_status,
 	} ) ),
 	withDispatch( ( dispatch ) => ( {
 		isActive: dispatch( 'core/editor' ).editPost( {
 			meta: {
-				[ '_coil_monetize_post_status' ]: isActive
```
Comment (JoblersTune) [inline src/extensions/components/monetize-mode.js:40]: And leave this as it was before in develop as well.

**Review by JoblersTune — CHANGES_REQUESTED** [review]
If we can just revert those two lines of code regarding isActive in the  src/extensions/components/monetize-mode.js file that I commented on then all looks good!

**Review by JoblersTune — APPROVED** [review]

**Review by marcperel — COMMENTED** [review]
👍

**Pull request merged**